Gemfile

```diff
@@ -2,4 +2,7 @@
 
 source "https://rubygems.org"
 
-# gem "rails"
+gem "pg"
+gem "rspec"
+gem "simplecov"
+gem "bcrypt"
```

Comment:
read about this symbol and learn how to get rid of this.

Reply:
which symbol?

Reply:
Red symbol which is coming at the end of file